OxfordBodleian/Juel-Jensen/BDLaethg38.xml

```diff
@@ -50,60 +108,154 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    Some letters are provided with loops at their ends, like <foreign xml:lang="gez">ፓ
+                                        (<locus target="#1r30"/>), 
+                                        <foreign xml:lang="gez">ሐ</foreign>/<foreign xml:lang="gez">ሑ</foreign> (e.g. <locus target="#1r56 #1r57"/>), and 
+                                            <foreign xml:lang="gez">ዎ</foreign> (<locus target="#2r4"/>).
+                                    These letters resemble Brillenbuchstaben <!--italics-->, but they are used within words in the body of the text.</desc>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">Few lines of the incipits and other lines within the text, the name of Mary, some of the Brillenbuchstaben <!--italics-->.</seg>
                             </handNote>
                         </handDesc>
                         
-                       <!-- <decoDesc>
-                         
-                        </decoDesc>-->
-                        
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>The verso is blank.</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc>Scribal corrections or interventions are very rare. They consist of letters crossed out or added interlineally.</desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>The name of the owner, mentioned in supplications throughout and introduced by the epithet <foreign xml:lang="gez">አመትከ፡</foreign> "your maid", was <persName ref="PRS13973WalattaMa"/>.
+                                        The name is partially illegible due to faded ink. Also, some of the letters of the name are very often crossed out and replaced with those of 
+                                        another poorly decipherable name.</desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <desc>Brillenbuchstaben <!--italics--> are incorporated within the text into several places of <ref target="#ms_i2 #ms_i3 #ms_i5">textual units II, III, and V</ref>.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
                         
+                        <bindingDesc>
+                            <binding>
+                                <decoNote xml:id="b1">The three parchment strips that compose the scroll are sewn together by means of narrow parchment strips.</decoNote>
```

Comment:
Perhaps, can we add (...) sewn together "with a running stitch"? (...)

Reply:
Thank you Eliana, modified.
Feel free to expand the binding part if you feel appropriate